packages/connect-explorer/src/js/data/methods/management/resetDevice.ts

```diff
@@ -0,0 +1,47 @@
+const name = 'resetDevice';
+const docs = 'methods/resetDevice.md';
```

Comment:
It looks like this variable `docs` is never used.

Reply:
fixed